### DISCUSSION TITLE
Is there any data collected by Kingfisher?

### DISCUSSION THREAD

**Discussion by stremsdoerfer** [onevcat/Kingfisher#1928] [Q&A] [answered]
Hi !

In order for us to provide accurate Privacy Nutrition Labels to Apple during our review, we need to know whether all our third party libraries collect any kind of data. Is there any data collected by kingfisher that we should specify in our review? (such as crash, usage, diagnostics etc.)

Thanks!

**Comment by onevcat** [comment, marked as answer]
No. Kingfisher does not collect anything from users and the devices.

**Reply by stremsdoerfer** [reply]
Awesome, thanks for your quick answer
